www/js/controllers/vehicle-map-controller.js

```diff
@@ -2,7 +2,7 @@ angular.module('pvta.controllers').controller('VehicleMapController', function (
   var bounds = new google.maps.LatLngBounds();
 
   var mapOptions = {
-    center: bounds.getCenter(),
+    center: new google.maps.LatLng(42.386270, -72.525844),
```

Comment:
I don't know how that happened on this branch, I made that change on my walking directions branch